runtime/src/snapshot_utils.rs

```diff
@@ -1536,7 +1536,7 @@ pub fn snapshot_bank(
     archive_format: &ArchiveFormat,
     hash_for_testing: Option<Hash>,
 ) -> Result<()> {
-    let storages: Vec<_> = root_bank.get_snapshot_storages();
+    let storages = root_bank.get_snapshot_storages();
```

Comment:
Mainly as an fyi, and not the biggest deal in the world, but when we get_snapshot_storages more deeply in AccountsDb, we also return a corresponding vector of Slots. It turns out to be expensive to ask each storage for its slot. Definitely would tackle this in a different pr. More of an optimization possibility for later.